core/templates/dev/head/pages/community-dashboard-page/community-dashboard-page.directive.html

```diff
@@ -29,6 +29,9 @@
       </div>
     </div>
   </div>
+  <div ng-if="$ctrl.activeTabName === 'questionTab'">
+    <skill-opportunities></skill-opportunities>
```

Comment:
Maybe? Though the opportunities are tied to a specific skill as well, and we would need to update all the references.

Reply:
This will help us to create a directive similar to its tab name.

Also, if you'll see we have Exploration opportunity models but we fetch translation and voiceover opportunities out of it and have translation-opportunities directive.